app.coffee

```diff
@@ -6,8 +6,13 @@ http = require("http")
 Settings = require('settings-sharelatex')
 logger = require('logger-sharelatex')
 logger.initialize("document-updater")
+
+serializers = require("./app/js/LoggerSerializers")
+serializers.apply(logger)
+
 logger.logger.serializers.docs = require("./app/js/LoggerSerializers").docs
```

Comment:
I think these lines for docs and files can be deleted now.
Also let's use the `addSerializers` method as that is what is documented by bunyan.

Reply:
How's this?

Reply:
Great.